React/Modules/RCTServerUtils.m

```diff
@@ -0,0 +1,21 @@
+
+#import "RCTServerUtils.h"
+#import "RCTIPAddress.h"
+
+@implementation RCTServerUtils
+
++ (NSURL *)getServerURL:(NSString *)path
```

Comment:
Nit: this name is not very ObjC-like, prefer something like: `serverURLForPath:`